tools/deploy_assisted_installer.py

```diff
@@ -53,6 +53,8 @@ def main():
             data["spec"]["template"]["spec"]["containers"][0]["imagePullPolicy"] = "Never"
         else:
             data["spec"]["template"]["spec"]["containers"][0]["imagePullPolicy"] = "Always"
+        if deploy_options.target == utils.OCP_TARGET:
+            data["spec"]["template"]["spec"]["containers"][0]["env"].append({'name':'DEPLOY_TARGET', 'value': "onprem"})
```

Comment:
this should be:
data["spec"]["template"]["spec"]["containers"][0]["env"].append({'name':'DEPLOY_TARGET', 'value': "ocp"})

Reply:
ok, I see it's already added to DeployTarget switch/case[*], so could be changed indeed.
[*] https://github.com/openshift/assisted-service/blob/6ea925a3300d88831cd90e4af5dd925a90ebb2b8/cmd/main.go#L218